### PR TITLE
  Add documentation for interfacing with Objective-C

### PR DESCRIPTION
**Pull request by jacob-carlborg**
See https://github.com/D-Programming-Language/dlang.org/pull/1039 for reference and comments.

### CODE REVIEW

**Comment by jacob-carlborg** [comment]
@CyberShadow

**Comment by CyberShadow** [comment]
Thanks!

File: attribute.dd [hunk 1]
```diff
@@ -91,6 +92,9 @@ $(GNAME LinkageType):
         $(D C) and $(D D) must be supplied, the others are what
         makes sense for the implementation.
         $(D C++) offers limited compatibility with C++.
+        $(D Objective-C) offers limited compatibility with Objective-C,
+        see the $(LINK2 objc_interface.html, Interfacing to Objective-C)
```
Comment (MartinNowak) [inline attribute.dd]: Guess this should have used `$(ROOT_DIR)objc_interface.html`.

Comment (jacob-carlborg) [inline attribute.dd]: Why is that? The link works. Or is for an other format?

**Pull request merged**